common/models/types.ts

```diff
@@ -53,6 +54,10 @@ export type ClientInfo = {
 	status?: PlayerStatus;
 };
 
+export type SegmentCategories = {
+	[K in Category]: boolean;
+}
```

Comment:
It would be slightly more efficient disk space wise if this type was an array of `Category` instead of an object. Maybe not so much in postgres (because of jsonb), but definitely in redis. On the other hand, this approach doesn't require us to use `Set` to guarantee uniqueness of items, which would add to the complexity.

What's your opinion?

Reply:
Totally agree. I completely forgot about the redis layer.